### PR TITLE
fixed local change link

### PR DESCRIPTION
**Pull request by naveed-ahmad**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32792

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32793

File: src/helpers/setLocal.js [hunk 1]
```diff
@@ -26,5 +26,5 @@ export default (req) => {
 
   const localeData = require('../locale/' + currentLocal + '.js'); // eslint-disable-line
 
-  return localeData.messages;
+  return localeData.default.messages;
```
Comment (naveed-ahmad) [inline src/helpers/setLocal.js]: I'm not sure what has changed in recent react upgrade, and if that's best way to imports files. @mmahalwy please suggest the better solution ;)

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32795

File: src/helpers/setLocal.js (hunk 1 above)
Comment (mmahalwy) [inline src/helpers/setLocal.js]: Yes, you need to use `.default` now that we are on webpack 2. We had a plugin that did it automatically and broke tree-shaking of webpack 2.

**Comment by mmahalwy** [comment]
@naveed-ahmad check travis please. you have a lint error.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32797

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32798

**Pull request merged**